qiskit/chemistry/components/bosonic_basis/bosonic_basis.py

```diff
@@ -0,0 +1,38 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+""" Bosonic Basis """
+
+from typing import List, Tuple
+
+
+class BosonicBasis:
+    """ Basis to express a second quantization Bosonic Hamiltonian. """
+
+    def run(self, threshold: float = 1e-6) \
```

Comment:
Is there a better name we can give this than 'run' - since it maps against a basis would 'map' be better, or 'convert' for example....

Reply:
yes we can use convert